app/assets/stylesheets/rad_common/rad_common.css

```diff
@@ -259,3 +268,92 @@ td.fa.security-group-td { border-style: none; }
     margin-bottom: 0;
   }
 }
+
```

Comment:
Enhance the comments on this section so we have more insights when looking back.